bin/ferver

```diff
@@ -9,6 +9,10 @@ OptionParser.new do |opts|
   opts.banner = 'Ferver: A simple web app to serve files over HTTP. Version: ' + Ferver::VERSION
   opts.separator ''
 
+  opts.on('-i', '--invisible', 'Serve invisible files') do |i|
```

Comment:
Yes - that's a better name.  What's the short switch though? Can't be -h - how about -a for serve all files  ( as used by list command  ls -a ) ?